packages/changed-elements-react/src/contentviews/PropertyComparisonTable.tsx

```diff
@@ -30,6 +30,8 @@ export interface PropertyComparisonTableProps {
   isSideBySide?: boolean | undefined;
 
   onSideBySideToggle?: (() => void) | undefined;
+
+  displaySideBySideToggle?: boolean;
```

Comment:
Let's add `| undefined` union so that projects using `"exactOptionalPropertyTypes": "true"` in their tsconfig.json